services/graphite/parser_test.go

```diff
@@ -602,6 +602,27 @@ func TestApplyTemplateSpecific(t *testing.T) {
 	}
 }
 
+// Test that most specific template is N/A
```

Comment:
Ahem, the first match should be applied in this case. However, it goes deeper to find a more specific one, but cannot find it. The original result will roll back to the global default template, not the first match.